deploy.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Deployer;
+
+require 'recipe/common.php';
+
+/*
+ * REDAXO Configuration
+ */
+
+$baseDir = dirname(dirname(dirname(dirname(__DIR__))));
+if (0 === strpos($baseDir, getcwd())) {
+    $baseDir = substr($baseDir, strlen(getcwd()));
+    $baseDir = ltrim($baseDir.'/', '/');
+}
+
+set('shared_dirs', [
```

Comment:
Hmm, ich finde gerade auf deren Seite auch nirgends eine wirkliche Erklärung.
Am ehesten hier: https://deployer.org/docs
Suchen nach "On first Deployer will create the following directories on the server".

Die shared-dirs werden zentral in einen `shared`-Ordner gelegt, und in den einzelnen Releases werden dann nur Symlinks angelegt.
Die Shared-Dirs "überstehen" also neue Releases.